alibi/datasets/default.py

```diff
@@ -67,7 +66,45 @@ def fetch_imagenet_10(url_id: int = 0) -> Dict:
     except RequestException:
         logger.exception("Could not connect, URL may be out of service")
         raise
-    imagenet10 = joblib.load(urllib.request.urlopen(url))
+    tar = tarfile.open(fileobj=BytesIO(resp.content), mode="r:gz")
+    imagenet10 = {}
+
+    int_to_str_labels = json.load(tar.extractfile('imagenet10/int_to_str_labels.json'))  # type: ignore
```

Comment:
Can you narrow down the `type: ignore` statements using the error codes shown please? I.e. `type: ignore[error-code]`.